src/rtnl.rs

```diff
@@ -130,7 +130,8 @@ pub struct Ifinfomsg {
     pub ifi_index: libc::c_int,
     /// Interface flags
     pub ifi_flags: Vec<Iff>,
-    ifi_change: libc::c_uint,
+    /// Change mask
+    pub ifi_change: libc::c_uint,
```

Comment:
I think it's best to convert this to the type `Iff` for now. I have a way to mark multiple types as usable in a field in the `consts` infrastructure if someone comes along and requests that other values should be supported. Until then, I think it's safe to assume that `Iff` is a reasonable accepted type.

Reply:
Done